src/components/SingleReviewPage.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import SingleReviewCard from "./SingleReviewCard";
+import CommentCard from "./CommentCard";
+
+const SingleReviewPage = () => {
+  const { review_id } = useParams();
+  const [review, setReview] = useState(null);
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetch(`https://ncgamesapp.herokuapp.com/api/reviews/${review_id}`)
+      .then((res) => {
+        return res.json();
+      })
+      .then(({ review }) => {
+        setReview(review);
+      })
+      .then(
+        fetch(
```

Comment:
we are chaining the action of getting the review and then waiting for that before we get the comments, is this something you want to do, it feels a bit like blocking code